--- a/src/App.tsx
+++ b/src/App.tsx
@@ -26,7 +26,7 @@
 import PagosListPage from './pagos/pages/PagosListPage';
 import PasajeroCreatePage from './pasajeros/pages/PasajeroCreatePage';
 import ReinscripcionesListPage from './reinscripciones/pages/ReinscripcionesListPage';
-import ReinscripcionCreatePage from './reinscripciones/pages/ReinscripcionCreatePage';
+// import ReinscripcionCreatePage from './reinscripciones/pages/ReinscripcionCreatePage';
 
 export default function App() {
   return (
@@ -43,7 +43,7 @@
           <Route path="/pagos" element={<PagosListPage />} />
           <Route path="/pasajeros/crear" element={<PasajeroCreatePage />} />
           <Route path="/reinscripciones" element={<ReinscripcionesListPage />} />
-          <Route path="/reinscripciones/crear" element={<ReinscripcionCreatePage />} />
+          {/* <Route path="/reinscripciones/crear" element={<ReinscripcionCreatePage />} /> */}
         </Route>
       </Routes>
     </Router>
